dependency-check-maven-config.xml

```diff
@@ -185,6 +186,16 @@
         <cve>CVE-2017-3156</cve>
         <cve>CVE-2016-6812</cve>
     </suppress>
+    <suppress>
+        <notes>
+            This CVE applies to Camel versions 2.16.0 through 2.18.1.
+            DDF version was changed to 2.16.0 and OWASP thinks a custom bundle with the word “camel”
+            means DDF uses version 2.16.0 of Camel. This is not true. At the time of this writing,
+            DDF uses Camel version camel 2.22.1 (see root POM file), which is not subject to this
+            vulnerability.
+        </notes>
+        <cve>CVE-2016-8749</cve>
```

Comment:
Is this only necessary for master?

Reply:
Sigh. There shouldn't have been changes to this file in the commit. Let me back it out.